docs/resources/ec_deployment.md

```diff
@@ -105,6 +105,9 @@ The following arguments are supported:
 
 * `deployment_template_id` - (Required) Deployment template identifier to create the deployment from. See the [full list](https://www.elastic.co/guide/en/cloud/current/ec-regions-templates-instances.html) of regions and deployment templates available in ESS.
 * `version` - (Required) Elastic Stack version to use for all the deployment resources.
+
+~> Using an Elastic Stack version prior to `6.6.0` is not supported.
```

Comment:
I think this is important enough that it should go closer to the beginning. Around line 10 before the note about traffic filters would be good I think!

```suggestion
~> **Note on Elastic Stack versions** Using a version prior to `6.6.0` is not supported.
```

Reply:
sounds good